### PR TITLE
Improve the Contact Bot page

### PR DESCRIPTION
**Pull request by sbis04**
Have made some improvements to the **Contact Bot** page under `Tutorials > Examples > Basic`.

The page looks like this:

Contact Bot page

Tag #891

### CODE REVIEW

**Review by shwetalsoni — APPROVED** [review]

**Review by proudofsimin — APPROVED** [review]

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Comment by sbis04** [comment]
@Rohitesh-Kumar-Jain I guess this qualifies to be added to #1056

**Comment by Rohitesh-Kumar-Jain** [comment]
> @Rohitesh-Kumar-Jain I guess this qualifies to be added to #1056

Yes I will add it there : )

File: docusaurus/docs/examples/basic/contact-bot.mdx [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+
+Let's get started with building the Wechaty Contact bot using Wechaty.
+
+### 1. Initialize project
+
+Create a new folder called `contact-bot` and move into that directory:
+
+```bash
+mkdir contact-bot
+cd contact-bot
```
Comment (lijiarui) [inline docusaurus/docs/examples/basic/contact-bot.mdx:133]: It looks a little bit strange that you already create ` wechaty-getting-started` before, and now create another folder `contact-bot` here, but they are the same code.

```
git clone https://github.com/wechaty/wechaty-getting-started.git
cd wechaty-getting-started
```

File: docusaurus/docs/examples/basic/contact-bot.mdx [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+
+1. If you want to use **WhatsApp**, install `wechaty-puppet-whatsapp`:
+
+  ```bash
+  npm install wechaty-puppet-whatsapp
+  ```
+2. If you want to use **WeChat**, install `wechaty-puppet-wechat`:
+
+  ```bash
+  npm install wechaty-puppet-wechat
```
Comment (lijiarui) [inline docusaurus/docs/examples/basic/contact-bot.mdx]: There is a lot of puppets can use for WeChat, here is some information for you, you can change this in your way into the doc:

If you want to use **[WeChat](https://www.wechat.com/)**, you can try the following puppet:
- Web Protocol： `wechaty-puppet-wechat`
- iPad Protocol:
  -  padlocal: `wechaty-puppet-padlocal`: `npm install wechaty-puppet-padlocal` and get a token like `puppet_padlocal_XXX`, see more [puppet service padlocal](http://wechaty.js.org/docs/puppet-services/padlocal)
  - paimon: `wechaty-puppet-service`: `npm install wechaty-puppet-service` and get a token like `puppet_paimon_XXX`, see more [puppet service paimon](http://wechaty.js.org/docs/puppet-services/paimon)

File: docusaurus/docs/examples/basic/contact-bot.mdx [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+  ```
+
+3. If you want to use **WeCom**, install `wechaty-puppet-wxwork`:
+
+  ```bash
+  npm install wechaty-puppet-wxwork
```
Comment (lijiarui) [inline docusaurus/docs/examples/basic/contact-bot.mdx]: wxwork is using wechaty-puppet-service, so you should change this to the following:

If you want to use **[WeCom](https://work.weixin.qq.com/)**, you can install `wechaty-puppet-service`, and get a token like `puppet_wxwork_XXXXX`, 

see more [puppet-service wxwork](http://wechaty.js.org/docs/puppet-services/wxwork)

```
npm install wechaty-puppet-service
```

File: docusaurus/docs/examples/basic/contact-bot.mdx [hunk 4]
```diff
@@ -0,0 +1,377 @@
...
+
+```bash
+export WECHATY_LOG=verbose
+export WECHATY_PUPPET=wechaty-puppet-wechat
+
+# For using WhatsApp:
+# export WECHATY_PUPPET=wechaty-puppet-whatsapp
+
+# For using WeCom:
+# export WECHATY_PUPPET=wechaty-puppet-wxwork
```
Comment (lijiarui) [inline docusaurus/docs/examples/basic/contact-bot.mdx]: Should change to

```
export WECHATY_PUPPET=wechaty-puppet-service
export WECHATY_PUPPET_SERVICE_TOKEN="puppet_wxwork_XXXXX"
```

Get token from [puppet service wxwork](http://wechaty.js.org/docs/puppet-services/wxwork)

File: docusaurus/docs/examples/basic/contact-bot.mdx [hunk 5]
```diff
@@ -0,0 +1,377 @@
...
```
Comment (lijiarui) [inline docusaurus/docs/examples/basic/contact-bot.mdx]: Should change to

```
export WECHATY_PUPPET=wechaty-puppet-service
export WECHATY_PUPPET_SERVICE_TOKEN="puppet_wxwork_XXXXX"
```

Get token from [puppet service wxwork](http://wechaty.js.org/docs/puppet-services/wxwork)

File: docusaurus/docs/examples/basic/contact-bot.mdx [hunk 6]
```diff
@@ -0,0 +1,377 @@
...
+
+```bash
+set WECHATY_LOG=verbose
+set WECHATY_PUPPET=wechaty-puppet-wechat
+
+# For using WhatsApp:
+# export WECHATY_PUPPET=wechaty-puppet-whatsapp
+
+# For using WeCom:
+# export WECHATY_PUPPET=wechaty-puppet-wxwork
```
Comment (lijiarui) [inline docusaurus/docs/examples/basic/contact-bot.mdx]: Should change to

```
export WECHATY_PUPPET=wechaty-puppet-service
export WECHATY_PUPPET_SERVICE_TOKEN="puppet_wxwork_XXXXX"
```

Get token from [puppet service wxwork](http://wechaty.js.org/docs/puppet-services/wxwork)

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: docusaurus/docs/examples/basic/contact-bot.mdx (hunk 1 above)
Comment (sbis04) [reply docusaurus/docs/examples/basic/contact-bot.mdx:133]: Actually, those are two separate sections.

In **Getting started** section, it guides the user to clone a pre-made bot and run it locally on their system.

In **Building the bot** section, it guides the user step-by-step on how to build the bot from scratch.

If you see the bot building tutorials, we have followed a similar pattern in most of them.

File: docusaurus/docs/examples/basic/contact-bot.mdx (hunk 2 above)
Comment (sbis04) [reply docusaurus/docs/examples/basic/contact-bot.mdx]: Have made these changes.

File: docusaurus/docs/examples/basic/contact-bot.mdx (hunk 3 above)
Comment (sbis04) [reply docusaurus/docs/examples/basic/contact-bot.mdx]: Have made these changes.

File: docusaurus/docs/examples/basic/contact-bot.mdx (hunk 4 above)
Comment (sbis04) [reply docusaurus/docs/examples/basic/contact-bot.mdx]: Have made the changes.

File: docusaurus/docs/examples/basic/contact-bot.mdx (hunk 5 above)
Comment (sbis04) [reply docusaurus/docs/examples/basic/contact-bot.mdx]: Have made the changes.

File: docusaurus/docs/examples/basic/contact-bot.mdx (hunk 6 above)
Comment (sbis04) [reply docusaurus/docs/examples/basic/contact-bot.mdx]: Have made the changes.

Comment (sbis04) [reply docusaurus/docs/examples/basic/contact-bot.mdx]: Have made the changes.

**Comment by sbis04** [comment]
@lijiarui I have made the suggested changes.

File: docusaurus/docs/examples/basic/contact-bot.mdx [hunk 7]
```diff
@@ -0,0 +1,404 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
+```
+
+### 2. Install dependencies
+
```
Comment (lijiarui) [inline docusaurus/docs/examples/basic/contact-bot.mdx:160]: I thinks we should install wechaty first.

```
npm install wechaty
```

**Review by lijiarui — APPROVED** [review]
Just one more change, then I will merge this PR!

File: docusaurus/docs/examples/basic/contact-bot.mdx (hunk 7 above)
Comment (sbis04) [reply docusaurus/docs/examples/basic/contact-bot.mdx:160]: Have made the change.

**Comment by sbis04** [comment]
@lijiarui I have made the change.

**Review by lijiarui — APPROVED** [review]

**Pull request merged**